.github/docker/Dockerfile.gorgone-testing-bookworm

```diff
@@ -0,0 +1,31 @@
+FROM debian:bookworm
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Set locale
+RUN apt-get update && \
+    apt-get install -y locales libcurl4-openssl-dev curl wget zstd jq lsb-release mariadb-client iproute2 && \
+    apt-get update && \
+    apt-get install -y ca-certificates apt-transport-https software-properties-common gnupg2 && \
+    localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+ENV LANG=en_US.UTF-8
+
+# Install required packages and Robotframework
+RUN apt-get update && \
+    apt-get install -y \
+    python3 \
+    python3-dev \
+    python3-pip && \
+    pip3 install --break-system-packages --no-cache-dir \
+    robotframework \
+    robotframework-examples \
+    robotframework-databaselibrary \
+    pymysql \
+    robotframework-requests && \
+    echo "deb https://packages.centreon.com/apt-standard-24.04-stable/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/centreon.list && \
```

Comment:
If using develop, this may not work since there is currently no repository for debian based on develop (also considering that, the version number is increasing every months)
for this to work, you should use apt-standard-24.05-testing o apt-standard-24.05-unstable

Reply:
@tuntoja For now we can't rebuild all dependency of gorgone on each tests (it's too long and most of the dependancy are in centreon-plugin anyways)
So we use the last stable repository to download all dependency. There will be problem when we change some dependency of gorgone, but they don't change often.